samples/bluetooth/direction_finding_connectionless_rx/boards/nrf52833dk_nrf52833.conf

```diff
@@ -0,0 +1,18 @@
+#
+# Copyright (c) 2021 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+CONFIG_BT_CTLR=y
+CONFIG_BT_LL_SW_SPLIT=y
+
+CONFIG_BT_CTLR_ADV_EXT=y
+CONFIG_BT_CTLR_SYNC_PERIODIC=y
+
+# Enable Direction Finding Feature including AoA and AoD
+CONFIG_BT_CTLR_DF=y
+CONFIG_BT_CTLR_DF_ANT_SWITCH_TX=n
```

Comment:
Shouldn't that be under line 17 for clarity ?

Reply:
Moved below line `# Enable Direction Finding Feature including AoA and AoD`